README.md

```diff
@@ -1 +1,46 @@
-# bff-samples
\ No newline at end of file
+## Ballerina Persists module with databases
```

Comment:
Why did we update the repo README.md with the details of this sample? Ideally we need to add a separate README.md within sample folder.

Reply:
updated